homeassistant/components/tasmota/sensor.py

```diff
@@ -54,11 +54,18 @@
 
 # A Tasmota sensor type may be mapped to either a device class or an icon, not both
 SENSOR_DEVICE_CLASS_ICON_MAP: dict[str, dict[str, Any]] = {
+    hc.SENSOR_ACTIVE_ENERGYEXPORT: {DEVICE_CLASS: SensorDeviceClass.ENERGY},
+    hc.SENSOR_ACTIVE_ENERGYIMPORT: {DEVICE_CLASS: SensorDeviceClass.ENERGY},
```

Comment:
This should have a state class, I guess state class TOTAL, with or without last_reset: https://developers.home-assistant.io/docs/core/entity/sensor/#how-to-choose-state_class-and-last_reset